### ISSUE TITLE
No collections being shown

### ISSUE THREAD

**Issue by neil-s** [msavin/Mongol#154]
I just installed Mongol on a reasonably new project, and don't see my collections listed when I hit Ctrl+M, just an empty widget saying I'm not signed in. The collections should be discovered automatically without me explicitly adding them, right?

Here is my .meteor/packages:
```

# Meteor packages used by this project, one per line.
# Check this file (and the other files in this directory) into your repository.
#
# 'meteor add' and 'meteor remove' will edit this file for you,
# but you can also edit it by hand.

meteor-base@1.3.0             # Packages every Meteor app needs to have
mobile-experience@1.0.5       # Packages for a great mobile UX
mongo@1.4.2                   # The database Meteor supports right now
blaze-html-templates@1.0.4 # Compile .html files into Meteor Blaze views
reactive-var@1.0.11            # Reactive variable for tracker
tracker@1.1.3                 # Meteor's client-side reactive programming library

standard-minifier-js@2.3.1    # JS minifier run for production mode
es5-shim@4.7.0                # ECMAScript 5 compatibility for older browsers
ecmascript@0.10.0              # Enable ECMAScript2015+ syntax in app code
shell-server@0.3.1            # Server-side component of the `meteor shell` command

autopublish@1.0.7             # Publish all data to the clients (for prototyping)
insecure@1.0.7                # Allow all DB writes from clients (for prototyping)
juliancwirko:postcss
less
jquery
semantic:ui
aldeed:collection2@3.0.0
accounts-ui
accounts-facebook
facebook-config-ui
accounts-google
google-config-ui
session
msavin:mongol
```

And here is my .meteor/versions

```
accounts-base@1.4.2
accounts-facebook@1.3.1
accounts-google@1.3.1
accounts-oauth@1.1.15
accounts-ui@1.3.0
accounts-ui-unstyled@1.4.0
aldeed:collection2@3.0.0
allow-deny@1.1.0
autopublish@1.0.7
autoupdate@1.4.0
babel-compiler@7.0.2
babel-runtime@1.2.2
base64@1.0.10
binary-heap@1.0.10
blaze@2.3.2
blaze-html-templates@1.1.2
blaze-tools@1.0.10
boilerplate-generator@1.4.0
caching-compiler@1.1.11
caching-html-compiler@1.1.2
callback-hook@1.1.0
check@1.3.0
dburles:mongo-collection-instances@0.3.1
ddp@1.4.0
ddp-client@2.3.1
ddp-common@1.4.0
ddp-rate-limiter@1.0.7
ddp-server@2.1.2
deps@1.0.12
diff-sequence@1.1.0
dynamic-import@0.3.0
ecmascript@0.10.2
ecmascript-runtime@0.5.0
ecmascript-runtime-client@0.6.1
ecmascript-runtime-server@0.5.0
ejson@1.1.0
es5-shim@4.7.3
facebook-config-ui@1.0.1
facebook-oauth@1.4.0
geojson-utils@1.0.10
google-config-ui@1.0.0
google-oauth@1.2.5
hot-code-push@1.0.4
html-tools@1.0.11
htmljs@1.0.11
http@1.4.0
id-map@1.1.0
insecure@1.0.7
jquery@1.11.11
juliancwirko:postcss@1.2.0
launch-screen@1.1.1
less@2.7.12
livedata@1.0.18
localstorage@1.2.0
logging@1.1.19
meteor@1.8.2
meteor-base@1.3.0
minifier-css@1.3.0
minifier-js@2.3.1
minimongo@1.4.3
mobile-experience@1.0.5
mobile-status-bar@1.0.14
modules@0.11.3
modules-runtime@0.9.2
mongo@1.4.3
mongo-dev-server@1.1.0
mongo-id@1.0.6
msavin:mongol@0.5.5
npm-mongo@2.2.34
oauth@1.2.1
oauth2@1.2.0
observe-sequence@1.0.16
ordered-dict@1.1.0
promise@0.10.1
raix:eventemitter@0.1.3
random@1.1.0
rate-limit@1.0.8
reactive-dict@1.2.0
reactive-var@1.0.11
reload@1.2.0
retry@1.1.0
routepolicy@1.0.12
semantic:ui@2.2.6_5
semantic:ui-data@2.2.6_4
server-render@0.3.0
service-configuration@1.0.11
session@1.1.7
shell-server@0.3.1
shim-common@0.1.0
socket-stream-client@0.1.0
spacebars@1.0.15
spacebars-compiler@1.1.3
standard-minifier-js@2.3.1
templating@1.3.2
templating-compiler@1.3.3
templating-runtime@1.3.2
templating-tools@1.1.2
tmeasday:check-npm-versions@0.3.2
tracker@1.1.3
ui@1.0.13
underscore@1.0.10
url@1.2.0
webapp@1.5.0
webapp-hashing@1.0.9
```

**Comment by msavin** [comment]
For some reason, your app is using an ancient version of Mongol `msavin:mongol@0.5.5`

It's at like 4.0.0 right now

**Comment by neil-s** [comment]
I noticed that, but for some reason Meteor thinks later versions will be incompatible with something else. I tried uninstalling and reinstalling Mongol, no dice. I tried `meteor update mongol` and it said "The specified packages are at their latest compatible versions.". Are there any packages on that list that you may have a dependency on or from?

**Comment by neil-s** [comment]
So the issue was twofold, leaving these details here in case someone else runs into it:

1) Meteor was installing an old version. I managed to get it to install a new version, which fixed the issue. I did some series of operations that involved adding and removing packages, deleting the versions file, reordering packages in the packages file. Not sure what shortest reproducible set of steps fixes the issue, but I believe it boils down to uninstalling aldeed:collection2 and installing aldeed:collection2-core. 

2) The old version was also displaying the collections, but it was white text on a white background, so I couldn't see them. Clicking underneath the visible Mongol window did click on the headers in the accordion and make the collections show up accordingly. This is no longer relevant with the new version however, which correctly applies its own black backdrop to the whole widget.
